docs-v2/integrations/all/hubspot.mdx

```diff
@@ -3,29 +3,146 @@ title: HubSpot
 sidebarTitle: HubSpot
 ---
 
-API configuration: [`hubspot`](https://nango.dev/providers.yaml)
+## Overview
 
-## Features
+<CardGroup cols={3}>
+<Card title="Pre-built tooling" icon="screwdriver-wrench" href="#pre-built-tooling"/>
+<Card title="Pre-built use-cases" icon="square-check" href="#pre-built-use-cases"/>
+<Card title="Pre-requisites" icon="triangle-exclamation" href="#pre-requisites"/>
+<Card title="Setup guide" icon="rocket" href="#setup-guide"/>
+<Card title="Useful links" icon="circle-info" href="#useful-links"/>
+<Card title="API gotchas" icon="biohazard" href="#api-gotchas"/>
+</CardGroup>
 
-| Features | Status |
-| - | - |
-| [Auth (OAuth)](/guides/getting-started/authorize-an-api-from-your-app) | ✅ |
-| [Sync data](/guides/getting-started/read-from-an-api) | ✅ |
-| [Perform workflows](/guides/getting-started/write-to-an-api) | ✅ |
-| [Proxy requests](/guides/proxy-requests-to-an-api) | ✅ |
-| [Receive webhooks](/guides/receive-webhooks-from-an-api) | ✅ |
+## Pre-built tooling
 
-<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+<AccordionGroup>  
+  <Accordion title="✅ Authorization">
+    | Tools | Status | 
+    | - | - | 
+    | Pre-built authorization (OAuth 2) | ✅ |
+    | Credentials auto-refresh | ✅ |
+    | Authorization input validation | ✅ |
+    | Pre-built authorization UI | ✅ |
+    | Custom authorization UI | ✅ |
+    | End-user authorization guidelines | ✅ |
+    | Invalid credentials detection | ✅ |
+  </Accordion>
+  <Accordion title="✅ Read & write data">
+    | Tools | Status | 
+    | - | - | 
+    | Pre-built use-cases | ✅ |
+    | API unification | ✅ |
+    | 2-way sync | ✅ |
+    | Webhooks from Nango on data modifications | ✅ |
+    | Real-time webhooks from 3rd-party API | ✅ |
+    | Proxy requests | ✅ |
+  </Accordion>
+  <Accordion title="✅ Observability & data quality">
+    | Tools | Status | 
+    | - | - | 
+    | HTTP request logging | ✅ |
+    | End-to-type type safety | ✅ |
+    | Data runtime validation | ✅ |
```

Comment:
@bastienbeurier Since we mention some platform level features here, IMO we could add:
- OpenTelemetry export
- Slack Alerts on errors
- Integration status API